src/ORM/Sluggable/SluggableSubscriber.php

```diff
@@ -40,12 +27,12 @@ public function loadClassMetadata(LoadClassMetadataEventArgs $eventArgs)
             return;
         }
 
-        if ($this->isSluggable($classMetadata)) {
+        if (is_subclass_of($classMetadata->getName(), 'Knp\DoctrineBehaviors\Model\Sluggable\SluggableInterface')) {
```

Comment:
Wouldn't you prefer basing yourself on the *Properties trait to trigger the mapping ? Using the interface might cause problems in case of inheritance (CF #317).

What do you think about the pattern I used in my PRs (e.g. https://github.com/KnpLabs/DoctrineBehaviors/pull/322/files#diff-9af172be101c124fe7673df513efc25aL296)